src/vs/workbench/browser/media/part.css

```diff
@@ -78,6 +78,7 @@
 	background-size: 16px;
 	background-position: center center;
 	background-repeat: no-repeat;
+	padding: 2px; /* Make sure view actions and container actions align */
```

Comment:
@benibenj isn't this a dangerous change when you have actions in a part and then a toolbar right above (e.g. in the title), the actions would not appear aligned anymore.

This can happen when you look at the layout controls in the custom title while there are actions right below in the editor. I tried to align these 2 locations perfectly.